lua/wire/server/wirelib.lua

```diff
@@ -52,9 +52,14 @@ local CurTime = CurTime
 
 -- helper function that pcalls an input
 function WireLib.TriggerInput(ent, name, value, ...)
-	if (not IsValid(ent) or not HasPorts(ent) or not ent.Inputs) then return end
+	if not IsValid(ent) or not HasPorts(ent) then return end
 
-	local input = ent.Inputs[name]
+	local entTbl = ent:GetTable()
```

Comment:
If we store a `local IsValid = FindMetaTable("Entity").IsValid local ent_GetTable = FindMetaTable("Entity").GetTable` and use those instead, even more __index calls can be reduced. (IsValid returns false if inputted nil as well so you don't need a nil check which is nice).